provider/oauth2/abstract_models.py

```diff
@@ -25,7 +25,7 @@
 except ImportError:
     timezone = None
 
-AUTH_USER_MODEL = getattr(settings, 'AUTH_USER_MODEL', 'auth.User')
+AUTH_USER_MODEL = getattr(settings, "AUTH_USER_MODEL", "auth.User")
```

Comment:
Black did most of the changes here. The only important one so far is adding `on_delete=CASCADE`